src/javascript/_common/utility.js

```diff
@@ -40,6 +40,41 @@ const getHighestZIndex = (selector = 'div,p,area,nav,section,header,canvas,aside
     return all.length ? Math.max(...all) : null;
 };
 
+const eu_countries = [
+    'Italy',
+    'Germany',
+    'France',
+    'Luxembourg',
+    'Greece',
+    'Saint Martin',
+    'Spain',
+    'Slovakia',
+    'Lithuania',
+    'Netherlands',
+    'Austria',
+    'Bulgaria',
+    'Slovenia',
+    'Cyprus',
+    'Belgium',
+    'Romania',
+    'Croatia',
+    'Portugal',
+    'Poland',
+    'Latvia',
+    'Estonia',
+    'Czech Republic',
+    'Finland',
+    'Hungary',
+    'Denmark',
+    'Sweden',
+    'Ireland',
+    'Isle of Man',
+    'Great Britain',
+    'Malta',
+];
+// check if client is from EU
+const isEuCountry = country => eu_countries.includes(country);
```

Comment:
we already have a `isEuCountry` function which dynamically decides it based on the API. why are we repeating this by hardcoding the countries?

Reply:
The checkbox has to be coded in a way, that it appears by selecting a eu country from the selectbox. The current isEuCountry function gets the countrycode if the user is registered or logged in, otherwise, it uses ip address locating to determine the countrycode. Ashkan mentioned that he doesn't want the checkbox to appear due to VPN/ipaddress, hence, it's hard coded. A similar function is used in DerivApp, and I have copied it here. 

I will definitely refactor the code a bit though, and give this function a different name, since isEuCountry already exists.